lib/analyzer/image-inspector.ts

```diff
@@ -76,7 +76,7 @@ async function pullWithDockerBinary(
       throw new Error(`Operating system is not supported`);
     }
 
-    if (err.stderr && err.stderr.includes("no matching manifest for")) {
+    if (err.stderr && err.stderr.includes("manifest unknown")) {
```

Comment:
if the error message has changed in recent versions, shouldn't we make sure we're also backwards compatible and support the old message as well?

Reply:
Good shout 👍🏻